kani-compiler/src/codegen_cprover_gotoc/codegen/intrinsic.rs

```diff
@@ -361,19 +361,9 @@ impl<'tcx> GotocCtx<'tcx> {
         macro_rules! codegen_size_align {
             ($which: ident) => {{
                 let tp_ty = instance.substs.type_at(0);
-                if tp_ty.is_generator() {
-                    let e = self.codegen_unimplemented_expr(
```

Comment:
Was this previously missed because sitting inside a macro made it hard to find references? Just curious.

Reply:
No, the macro had no effect on this as far as I can tell.